### PR TITLE
Clean logging and connection functions

### PR DESCRIPTION
**Pull request by Lerri-Cofannos**

### CODE REVIEW

File: example/.gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
-.env
\ No newline at end of file
+.env
+.venv_*
```
Comment (CallumHemsley) [inline example/.gitignore:2]: what is .venv and .pyc?

File: sdk-python/src/omnilog/send_to_db.py [hunk 2]
```diff
@@ -1,9 +1,19 @@
 import psycopg2
 
-def connect_to_db(url):
-    if not url:
```
Comment (CallumHemsley) [inline sdk-python/src/omnilog/send_to_db.py:4]: im not sure I understand why this has been taken out of the sdk

File: example/.gitignore (hunk 1 above)
Comment (Lerri-Cofannos) [reply example/.gitignore:2]: .venv_* is the folder hosting the python virtual environment and *.pyc are the cached python files

File: example/llm/get_openai_response.py [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
 
 OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
 
 if not OPENAI_API_KEY:
     raise EnvironmentError(f"OPENAI_API_KEY not found: {OPENAI_API_KEY}\nMake sure that you have an OPENAI_API_KEY value in the .env file.")
 
 openai.api_key = OPENAI_API_KEY
-def get_openai_response(prompt: str):
-    return openai.Completion.create(
+def get_openai_response(prompt: str) -> dict:
```
Comment (juskek) [inline example/llm/get_openai_response.py:13]: chefs kiss gotta love them typing

File: example/llm/get_openai_response.py [hunk 4]
```diff
@@ -1,15 +1,24 @@
 import os
+from dotenv import load_dotenv
```
Comment (juskek) [inline example/llm/get_openai_response.py:2]: Will this be removed when `send_to_db()` accepts database_url_args?

Comment (juskek) [reply example/llm/get_openai_response.py:2]: Oh ignore this, just realised it is in the example project

**Review by juskek — APPROVED** [review]

**Pull request merged**